fe1-web/src/features/wallet/objects/RollCallToken.ts

```diff
@@ -0,0 +1,64 @@
+import { Hash, KeyPairState, PopToken } from 'core/objects';
+
+export interface RollCallTokenState {
+  token: KeyPairState;
+  laoId: string;
+  rollCallId: string;
+  rollCallName: string;
+}
```

Comment:
I don't think I currently see the purpose of this class but I am probably missing something 😅 I think previously we did not store the token because we could rely on the deterministic generation based on the seed, why does that not work anymore?

Reply:
Oh wait, you do not actually store it but return it in a nicer format from `recoverWalletRollCallTokens`, right? Then we probably don't need `RollCallTokenState`, `fromState` and `toState`?

Reply:
Right, as I wasn't sure exactly how I was going to use those objects I defined a state anyways, but for now at least it is not of any use.